### PR TITLE
MMB: Show menu item in explore component

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/components/exploreCity/ExploreText.js [hunk 1]
```diff
@@ -17,32 +20,39 @@ const ExploreText = (props: Props) => {
...
+          ) : (
+            <Translation id="mmb.main_menu.explore_city.find_stuff_to_do" />
+          )}
+        </Text>
+      </View>
+      <View style={[styles.padding, styles.buttonContainer]}>
+        <ExploreButton cityName={city || ''} />
+      </View>
+      <SeparatorFullWidth />
+      <ExploreMenuGroup data={airport} />
```
Comment (RobinCsl) [inline app/MMB/src/components/exploreCity/ExploreText.js]: It seems like this should be concerning the airport of the first segment, e.g. on trip Vienna-Birmingham with segments Vienna-Dublin, Dublin-Birmingham, the explore menu group has button 
`Order Transport to DUB`.

Comment (tbergq) [reply app/MMB/src/components/exploreCity/ExploreText.js]: You are right, this should be arrival, not the actual leg. I will fix it tomorrow.

Comment (tbergq) [reply app/MMB/src/components/exploreCity/ExploreText.js]: I am unable to reproduce what you are mentioning. See screenshot. 
image

Comment (RobinCsl) [reply app/MMB/src/components/exploreCity/ExploreText.js]: And this is what I see in the real app:
image from ios

**Comment by tbergq** [comment]
This feature needs more refactoring, we will merge this PR as is for now.

**Pull request merged**